indico/modules/users/templates/favorites.html

```diff
@@ -10,13 +10,13 @@
     <div class="layout-wrapper">
         <div class="row">
             <div class="column col-50">
-                <div class="i-box titled">
+                <div class="i-box titled {{ 'no-padding' if not user.favorite_users }}">
```

Comment:
Usually they fit better in a context where you actually use expressions in the then/else part.

However, I just noticed that there's an [example doing it your way](http://jinja.pocoo.org/docs/dev/templates/#if-expression) in the jinja docs so I guess it's fine.